### ISSUE TITLE
WSS TLS_mgm related crash

### ISSUE THREAD

**Issue by hydrosine** [OpenSIPS/opensips#1087]
On initiating a connection (with JsSip webclient) to the WSS listener opensips crashes.

TLS_mgm is implemented the same as on opensips 2.2

```
Mar 30 15:25:23 srv-ams5-cod-wss-01 /usr/sbin/opensips[17397]: ERROR:proto_wss:tls_accept: New TLS connection from 84.25.96.29:11242 failed to accept
Mar 30 15:25:23 srv-ams5-cod-wss-01 /usr/sbin/opensips[17397]: ERROR:proto_wss:wss_read_req: cannot fix read connection
Mar 30 15:25:23 srv-ams5-cod-wss-01 kernel: [84426.601617] opensips[17397]: segfault at 0 ip 00007fb266fb9987 sp 00007fff71391568 error 4 in proto_wss.so[7fb266fb5000+1b000]

```

GDB output of the crash:
```
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `/usr/sbin/opensips -P /var/run/opensips.pid -w /home/crash'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007fb266fb9987 in wss_report (type=1, conn_id=7412088426354767098, conn_flags=6, extra=0x596285) at proto_wss.c:299
299     proto_wss.c: No such file or directory.
(gdb) bt full
#0  0x00007fb266fb9987 in wss_report (type=1, conn_id=7412088426354767098, conn_flags=6, extra=0x596285) at proto_wss.c:299
No locals.
#1  0x000000000053d305 in tcp_trigger_report (conn=conn@entry=0x7fb26a8d18f8, type=type@entry=1, extra=0x596285) at net/net_tcp_report.c:138
        job = 0x7fb26a8d18f8
        __FUNCTION__ = "tcp_trigger_report"
#2  0x0000000000539113 in handle_io (fm=<optimized out>, idx=idx@entry=0, event_type=event_type@entry=1) at net/net_tcp_proc.c:266
        ret = -1
        n = <optimized out>
        con = 0x7fb26a8d18f8
        s = 6
        rw = <optimized out>
        resp = <optimized out>
        response = {140404268538104, 1}
        __FUNCTION__ = "handle_io"
#3  0x000000000053c45f in io_wait_loop_epoll (h=<optimized out>, t=<optimized out>, repeat=<optimized out>) at net/../io_wait_loop.h:225
        ret = <optimized out>
        e = <optimized out>
        n = 1
        r = 0
#4  tcp_worker_proc_loop () at net/net_tcp_proc.c:355
        __FUNCTION__ = "tcp_worker_proc_loop"
#5  0x00000000005345e6 in tcp_start_processes (chd_rank=chd_rank@entry=0x7c8d40 <chd_rank>, startup_done=startup_done@entry=0x0) at net/net_tcp.c:1829
        r = 0
        reader_fd = {47, 50}
        pid = <optimized out>
        load_p = 0x7fb26a800df0
        __FUNCTION__ = "tcp_start_processes"
#6  0x000000000041c3a3 in main_loop () at main.c:683
        startup_done = 0x0
        chd_rank = 9
#7  main (argc=<optimized out>, argv=<optimized out>) at main.c:1283
        cfg_stream = <optimized out>
        c = <optimized out>
        r = <optimized out>
        tmp = 0x1 <error: Cannot access memory at address 0x1>
        tmp_len = <optimized out>
        port = <optimized out>
        proto = <optimized out>
        protos_no = <optimized out>
        options = 0x5678f8 "f:cCm:M:b:l:n:N:rRvdDFETSVhw:t:u:g:P:G:W:o:"
        ret = -1
        seed = 533157608
---Type <return> to continue, or q <return> to quit---
        __FUNCTION__ = "main"


```

**Comment by ionutrazvanionita** [comment]
Can you please print the line in frame 0? and if the line is
`if ( !*trace_is_on || (conn_flags & F_CONN_TRACE_DROPPED) )`
can you please print 
`p trace_is_on`?

**Comment by hydrosine** [comment]
Not quite sure what to do when you say. print the line in frame 0. Do you have some exact commands for me to execute?

> (gdb) p 0
> $1 = 0
> (gdb) p trace_is_on
> $2 = (int * const) 0x0
> (gdb)

**Comment by ionutrazvanionita** [comment]
It's ok I saw what I wanted to see. I just wanted you to call `f 0` to see the line in code where the crash happened, but it's obvious if trace_is_on is NULL. Did you update OpenSIPS to latest from github? This bug should have been fixed with https://github.com/OpenSIPS/opensips/commit/432dfb14df13735504c919ad4951a3b9f50d9824 on master and https://github.com/OpenSIPS/opensips/commit/c04098d89cb30720d1cc8166a7d17550c3803581 on 2.3 .
